vuepress/docs/next/docs/getting-started/README2.md

```diff
@@ -0,0 +1,411 @@
+---
```

Comment:
@jyunmitch what is README2? Accidental commit?

Reply:
Yes, accidental. I'll clean it up.